Test/Unit/Model/PaymentTest.php

```diff
@@ -327,10 +380,10 @@ private function mockApiResponse($path, $responseJSON) {
     }
 
     private function createPaymentMock() {
```

Comment:
All but 2 tests here call `createPaymentMock` - maybe it's worth throwing this in a `setUp`?